content/en/tracing/connect_logs_and_traces/python.md

```diff
@@ -52,9 +55,11 @@ hello()
 
 ### Without Standard Library Logging
 
-If you are not using the standard library `logging` module, you can use the `ddtrace.helpers.get_correlation_ids()` to inject tracer information into your logs. As an illustration of this approach, the following example defines a function as a *processor* in `structlog` to add `dd.trace_id` and `dd.span_id` to the log output:
+If you are not using the standard library `logging` module, you can use the `ddtrace.helpers.get_correlation_ids()` to inject tracer information into your logs.
+As an illustration of this approach, the following example defines a function as a *processor* in `structlog` to add tracer fields to the log output:
 
 ``` python
+from ddtrace import config
```

Comment:
```suggestion
import ddtrace
```

Reply:
context: Doing this since `from ddtrace import config` has a higher chance of colliding with a variable the name the customer is already using, so I typically just go for `import ddtrace` and then `ddtrace.tracer`/`ddtrace.config`, etc to "namespace" them under `ddtrace.`